### PR TITLE
File Slim/Http/Util.php

### PR DESCRIPTION
**Pull request by rndm2**
Cookie with '=' in value no more ignored by getCookie

### CODE REVIEW

File: Slim/Http/Util.php [hunk 1]
```diff
@@ -402,9 +402,10 @@ public static function parseCookieHeader($header)
         $headerPieces = preg_split('@\s*[;,]\s*@', $header);
         foreach ($headerPieces as $c) {
             $cParts = explode('=', $c);
```
Comment (mcuelenaere) [inline Slim/Http/Util.php:404]: Why not just change this to `$cParts = explode('=', $c, 2);` ?

Comment (codeguy) [inline Slim/Http/Util.php:404]: :+1: This is a cleaner solution. @rndm2 Do you want to make this change?

Comment (rndm2) [inline Slim/Http/Util.php:404]: Sorry, I have no time now.

Comment (codeguy) [inline Slim/Http/Util.php:404]: No worries. I'll do this and link to your PR for historical reference.

**Pull request closed without merging**